h2o-algos/src/main/java/hex/glm/GLMMetricBuilder.java

```diff
@@ -189,7 +189,7 @@ public final long resDOF() {
 
   protected void computeAIC(GLMModel gm) {
     if (gm._parms._calc_like && gm._finalScoring) { // uses likelihood which is calculated for the final scoring
-      _aic = -2 * _log_likelihood + 2 * Arrays.stream(gm.beta()).filter(b -> b != 0).count();
+      _aic = 2 * _log_likelihood + 2 * Arrays.stream(gm.beta()).filter(b -> b != 0).count();
```

Comment:
Can you add a R test using the parameter setup Tomas setup and check to make sure the AIC matches 87145.45?

> fitH2Otest2 <- h2o.glm(
+   remove_collinear_columns=TRUE,
+   training_frame = simDataH2O,
+   x = 'xt',
+   y = 'yr',
+   family = "tweedie",
+   link = "tweedie",
+   tweedie_variance_power = tweedie_p,
+   lambda = 0,
+   compute_p_values = p_valueAIC,
+   dispersion_parameter_method = "ml",
+   calc_like=TRUE,
+   fix_tweedie_variance_power = F,
+   tweedie_link_power = 0
+ )